### PR TITLE
Change the default prompt color from green to lime

### LINKED ISSUE #427: Make the shell prompt `$PS1` distinguishable from the host shell
Maybe we should change the color of `$PS1` by default

### PR DESCRIPTION
**Pull request by afbjorklund**
Closes #427

### CODE REVIEW

File: pkg/cidata/cidata.TEMPLATE.d/lima.env [hunk 1]
```diff
@@ -1,3 +1,4 @@
+LIMA_CIDATA_NAME={{ .Name }}
```
Comment (jandubois) [inline pkg/cidata/cidata.TEMPLATE.d/lima.env:1]: Please add this to the ""Environment variables" section in `docs/internal.md`

**Comment by afbjorklund** [comment]
Apparently this gives a warning, the second time you run it:

```
LIMA| WARNING: Failed to execute /mnt/lima-cidata/boot/08-shell-prompt.sh
```

```
patching file .bashrc
Reversed (or previously applied) patch detected!  Skipping patch.
2 out of 2 hunks ignored
```

Should probably leave some kind of text that could be grepped ?

File: pkg/cidata/cidata.TEMPLATE.d/lima.env (hunk 1 above)
Comment (afbjorklund) [reply pkg/cidata/cidata.TEMPLATE.d/lima.env:1]: Added

**Comment by AkihiroSuda** [comment]
Thanks, but this doen't seem to work with the default configuration of tmux (version 3.2a)?

(Adding `set-option -g default-terminal "screen-256color"` to `~/.tmux.conf` worked for me)

**Comment by afbjorklund** [comment]
Hmm, it shouldn't be any different from the previous (ubuntu) setup ? What is the tmux invocation for lima ?

**Comment by AkihiroSuda** [comment]
Maybe, just `PS1="[LIMA] $PS1"` might be fine

**Comment by afbjorklund** [comment]
And for UTF-8 it would be 🄻🄸🄼🄰 or something

**Comment by afbjorklund** [comment]
Then again, that will break a dozen _other_ scenarios - and doesn't even seem to work on GNOME Terminal

**Review by jandubois — APPROVED** [review]
LGTM

I probably would have appended the new logic at the end, overwriting the default prompt when needed. That would be somewhat more robust for future Ubuntu upgrades, that might break this patch. But we can also change to that approach if/when that happens.

**Pull request merged**